lib/jsutil.js

```diff
@@ -0,0 +1,32 @@
+'use strict'; // eslint-disable-line
+
+const debug = require('util').debuglog('jsutil');
+
+// JavaScript utility functions
+
+/**
+ * force @a func to be called only once, even if actually called
```

Comment:
I'm not sure if this is valid JSDoc syntax. In doxygen it means a link to some other object in the doxygen, so that func refers to the actual parameter. No problem to remove the `@a` if it does not make sense here.